ic10emu/src/vm.rs

```diff
@@ -378,21 +374,23 @@ impl VM {
             .get(&ic_id)
             .ok_or(VMError::UnknownIcId(ic_id))?
             .clone();
-        ic.borrow_mut().ic = 0;
+        ic.borrow().ic.replace(0);
         self.set_modified(id);
         for _i in 0..128 {
             if let Err(err) = ic.borrow_mut().step(self, ignore_errors) {
```

Comment:
Changing this to `borrow` instead of `borrow_mut` resolves the issue I previously mentioned

Reply:
fixed in the other PR